app/models/katello/glue/pulp/repo.rb

```diff
@@ -825,15 +825,15 @@ def full_path(smart_proxy = nil, force_https = false)
       pulp_uri = URI.parse(smart_proxy ? smart_proxy.url : SETTINGS[:katello][:pulp][:url])
       scheme   = (self.unprotected && !force_https) ? 'http' : 'https'
       if docker?
-        "#{pulp_uri.host.downcase}:5000/#{pulp_id}"
+        "#{pulp_uri.host.downcase}:5000/#{pulp_id}/"
```

Comment:
The docker cli doesn't like trailing slashes, and these aren't really browseable.  We should probably leave this one off